modules/guides/css/guides.in-page-navigation.css

```
@@ -0,0 +1,106 @@
+@media screen and (max-width: 767px) {
+  #guides__flex-wrapper {
+    position: fixed;
+    right: 0rem;
+    width: 100%;
+  }
+  #guides__in-page-nav {
+    clear: both;
+  }
+  #guides__open-mobile {
+    display: block;
+    float: right;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #guides__flex-wrapper {
+    min-width: 20%;
+    margin-right: 1rem;
+  }
+  #guides__in-page-nav.guides__in-page-nav--sticky {
```

You don't need to be this specific with the selection. We're using BEM to be specific enough with a single class already, so using `.guides__in-page-nav--sticky` is enough.
```suggestion
  .guides__in-page-nav--sticky {
```